### DISCUSSION TITLE
understanding-security-headers-part-3-cross-origin-embedder-policy/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#443] [Announcements]
# understanding-security-headers-part-3-cross-origin-embedder-policy/

In this post I discuss the Cross-Origin-Embedder-Policy, the requirements it places on your resources, and why it's required for some JavaScript APIs

https://andrewlock.net/understanding-security-headers-part-3-cross-origin-embedder-policy/

**Comment by OlegKarasik** [comment]
Thank you for the great series! Explanation of such complex topic worth a lot.

I think I have found a small typo (in the word “Resource” here):
> require-corp—the strictest value, requires that all cross-origin requests either include a **Cross-Origin-Reousurce-Policy** header (for no-cors requests) or a relevant Access-Control-Allow-Origin (ACAO) (for cors requests). More on that later!

**Comment by chrisxfire** [comment]
Thanks for this article, Mr. Lock.

Typo: *projections* vs *protections*:

> unsafe-none—the default value, doesn't add any additional projections.
